2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_setup.py:_flush():76] Current SDK version is 0.15.12
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_setup.py:_flush():76] Configure stats pid to 31813
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_setup.py:_flush():76] Loading settings from /Users/nigoo/.config/wandb/settings
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_setup.py:_flush():76] Loading settings from /Users/nigoo/Documents/Code/My_GitHub_Repositories/Text_Summarization_with_Transformers/wandb/settings
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_init.py:_log_setup():528] Logging user logs to /Users/nigoo/Documents/Code/My_GitHub_Repositories/Text_Summarization_with_Transformers/wandb/run-20231112_021629-aep4i6oy/logs/debug.log
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_init.py:_log_setup():529] Logging internal logs to /Users/nigoo/Documents/Code/My_GitHub_Repositories/Text_Summarization_with_Transformers/wandb/run-20231112_021629-aep4i6oy/logs/debug-internal.log
2023-11-12 02:16:29,459 INFO    MainThread:31813 [wandb_init.py:_jupyter_setup():474] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x1666829d0>
2023-11-12 02:16:29,460 INFO    MainThread:31813 [wandb_init.py:init():568] calling init triggers
2023-11-12 02:16:29,460 INFO    MainThread:31813 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {}
2023-11-12 02:16:29,460 INFO    MainThread:31813 [wandb_init.py:init():618] starting backend
2023-11-12 02:16:29,460 INFO    MainThread:31813 [wandb_init.py:init():622] setting up manager
2023-11-12 02:16:29,461 INFO    MainThread:31813 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-11-12 02:16:29,463 INFO    MainThread:31813 [wandb_init.py:init():628] backend started and connected
2023-11-12 02:16:29,470 INFO    MainThread:31813 [wandb_run.py:_label_probe_notebook():1276] probe notebook
2023-11-12 02:16:29,471 INFO    MainThread:31813 [wandb_run.py:_label_probe_notebook():1286] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-11-12 02:16:29,471 INFO    MainThread:31813 [wandb_init.py:init():720] updated telemetry
2023-11-12 02:16:29,488 INFO    MainThread:31813 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2023-11-12 02:16:30,026 INFO    MainThread:31813 [wandb_run.py:_on_init():2220] communicating current version
2023-11-12 02:16:30,216 WARNING MainThread:31813 [wandb_init.py:init():1192] interrupted
Traceback (most recent call last):
  File "/Users/nigoo/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1170, in init
    run = wi.init()
          ^^^^^^^^^
  File "/Users/nigoo/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 802, in init
    run._on_init()
  File "/Users/nigoo/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 2224, in _on_init
    version_result = version_handle.wait(timeout=30)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nigoo/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nigoo/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nigoo/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nigoo/anaconda3/lib/python3.11/threading.py", line 622, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nigoo/anaconda3/lib/python3.11/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2023-11-12 02:16:39,356 WARNING MsgRouterThr:31813 [router.py:message_loop():77] message_loop has been closed
